.github/workflows/python-ci.yml

```diff
@@ -94,10 +94,10 @@ jobs:
         os:
           - ubuntu
         looker:
-          - '21_6'
           - '21_8'
```

Comment:
Isn't 21.6 still the extended support release?

Reply:
oh interesting, I didn't realize the ESR pattern. According to https://docs.looker.com/relnotes/supported-releases 21.6 ESR support ends today (as does 21.8) so the list should actually only have 21_10, 21_12, 21_14